client/src/main/scala/drt/client/components/FlightTableRow.scala

```diff
@@ -135,10 +135,17 @@ object FlightTableRow {
 
       val expectedContent = maybeLocalTimeWithPopup(bestExpectedTime, Option(timesPopUp), None)
 
-      val charts = (flightWithSplits.hasValidApi, props.manifestSummary) match {
+      val charts = (flightWithSplits.hasApi, props.manifestSummary) match {
         case (true, Some(manifestSummary)) =>
-          <.div(^.className := "arrivals__table__flight-code__info",
-            FlightChartComponent(FlightChartComponent.Props(manifestSummary)))
+          val maybeLivePcpPax = flightWithSplits.apiFlight.bestPcpPaxEstimate(Seq(LiveFeedSource))
+          val maybePaxDiffAndPct = maybeLivePcpPax.map { pcpPax =>
+            val diff = pcpPax - manifestSummary.passengerCount
+            (diff, diff.toDouble / pcpPax)
+          }
+          if (maybePaxDiffAndPct.isEmpty || maybePaxDiffAndPct.exists(_._2 <= 1.05))
```

Comment:
will chart only be shown if within 5% threshold ?

Reply:
It will show for anything below 5% higher than the port operator figure